connectors/src/api/slack_channels_linked_with_agent.ts

```diff
@@ -152,20 +152,18 @@ export const patchSlackChannelsLinkedWithAgentHandler = withLogging(
   _patchSlackChannelsLinkedWithAgentHandler
 );
 
-type GetSlackChannelsLinkedWithAgentResBody =
-  | {
-      slackChannels: {
-        slackChannelId: string;
-        slackChannelName: string;
-        agentConfigurationId: string;
-      }[];
-    }
-  | APIErrorWithStatusCode;
+type GetSlackChannelsLinkedWithAgentResBody = WithConnectorsAPIErrorReponse<{
+  slackChannels: {
+    slackChannelId: string;
+    slackChannelName: string;
+    agentConfigurationId: string;
+  }[];
+}>;
 
 const _getSlackChannelsLinkedWithAgentHandler = async (
   req: Request<
     Record<string, string>,
-    { slackChannelIds: string[] } | APIErrorWithStatusCode,
+    { slackChannelIds: string[] },
```

Comment:
We should remove those, they are going against the definition of `Response<GetSlackChannelsLinkedWithAgentResBody>`

Reply:
done